org.aposin.licensescout.core/src/main/java/org/aposin/licensescout/mojo/AbstractScanMojo.java

```diff
@@ -291,13 +291,25 @@
     @Parameter(property = "resultDatabaseConfiguration", required = false)
     private DatabaseConfiguration resultDatabaseConfiguration;
 
+    /**
+     * Skips the execution.
+     */
+    @Parameter(defaultValue = "false", property = "skip", required = false)
```

Comment:
In org.aposin.licensescout.licensereport/pom.xml the skip parameter is set to false. Here the same parameter is marked as NOT required and defaults to false. My opinion is that OR we mark it as required OR we can remove it from the pom. What do you think?

Reply:
It was added in org.aposin.licensescout.licensereport/pom.xml for documentation purposes. This is the working example we refer people to as a starting point for including the LicenseScout into their own builds. Mentioning the parameter here demonstrates that the parameter exists, though in this case the value is technically not mandatory (the parameter should remain optional). So this is a service to programmers that don't read manuals.

Also note that it can be useful or even required to set the skip parameter to false despite this is the default value in a multi-module build where a parent POM globally sets it to true (thereby globally disabling the LicenseScout execution) and set it to false in certain projects to override the global value (thereby enabling the LicenseScout execution selectively). This is a common Maven pattern and will be explained in detail in the to-come LicenseScout cookbook.

But my compliment: you made a really clever point.